custom/icds/translations/integrations/transifex.py

```diff
@@ -70,19 +72,38 @@ def client(self):
         else:
             raise Exception(_("Transifex account details not available on this environment."))
 
+    @property
+    @memoized
+    def transifex_project_source_lang(self):
+        return self.client.transifex_lang_code(self.client.get_source_lang())
+
+    def _resource_name_in_project_lang(self, resource_slug):
+        """
+        return the name of the resource i.e module/form in source lang on Transifex
+
+        :param resource_slug: like module_moduleUniqueID
+        """
+        if MODULES_AND_FORMS_SHEET_NAME in resource_slug:
```

Comment:
Could this be changed to `if resource_slug == MODULES_AND_FORMS_SHEET_NAME:`? It seems like that would be more consistent other code in this PR.

Reply:
Its possible that resource_slug contains "_v12" in the end in case the translations files are kept based on version and so the check is a "in". The other way is to split first and then check. Can do that if that is better to understand.

Reply:
I see. So you could say `resource_slug.startswith(MODULES_AND_FORMS_SHEET_NAME + "_v")` and that would be less likely to cause a false positive, right? It's probably fine as is unless you think there is any chance that `resource_slug` could somehow contain `MODULES_AND_FORMS_SHEET_NAME` but not be the sheet you want to match.

Reply:
> somehow contain MODULES_AND_FORMS_SHEET_NAME but not be the sheet

Not possible. We control it. It can be either
module_moduleUniqueID(_v12) 
form_formUniqueID(_v12)
Module_and_forms(_v12)

Reply:
.. well.. even that step is controlled by HQ. 
This is not a user input.
The spreadsheet like structure is generated by HQ, [here](https://github.com/dimagi/commcare-hq/blob/42e5e59fb9f21d5161fe2460767eda81b2b820b5/corehq/apps/app_manager/app_translations/app_translations.py#L299-L298).

Reply:
So it's not an actual spreadsheet, just a dict that has a similar interface. I didn't realize that. Ignore what I said.